--- gnome-user-share-43.0/meson.build.orig	2023-05-18 21:34:36.569732449 +0200
+++ gnome-user-share-43.0/meson.build	2023-05-18 21:43:50.040067376 +0200
@@ -71,8 +71,12 @@ if systemd_systemduserunitdir == ''
   systemd_systemduserunitdir = dependency('systemd').get_variable(pkgconfig: 'systemduserunitdir')
 endif
 
+if get_option('httpd')[0] != 'httpd'
+  httpd_path = get_option('httpd')[0]
+else
 httpd = find_program(get_option('httpd'), required: false)
 httpd_path = (httpd.found() ? httpd.full_path() : '')
+endif
 
 sockets_deps = []
 if not cc.has_function('socket')
