--- gnome-user-share-48.0/meson.build.orig	2025-03-15 19:05:02.000000000 +0100
+++ gnome-user-share-48.0/meson.build	2025-04-30 21:28:11.050732923 +0200
@@ -55,6 +55,9 @@ if systemd_systemduserunitdir == ''
   endif
 endif
 
+if get_option('httpd')[0] != 'httpd'
+  httpd_path = get_option('httpd')[0]
+else
 httpd = find_program(get_option('httpd'), required: false)
 httpd_path = ''
 
@@ -76,6 +79,7 @@ if httpd.found()
     error('httpd version @0@ >= @1@: NO'.format(version, req_version))
   endif
 endif
+endif
 
 config_h.set_quoted('HTTPD_PROGRAM', httpd_path)
 
