--- gnome-user-share-47.0/meson.build.orig	2024-10-04 13:37:10.522107724 +0200
+++ gnome-user-share-47.0/meson.build	2024-10-04 18:14:25.841986467 +0200
@@ -79,6 +79,9 @@ if systemd_systemduserunitdir == ''
   systemd_systemduserunitdir = dependency('systemd').get_variable(pkgconfig: 'systemduserunitdir')
 endif
 
+if get_option('httpd')[0] != 'httpd'
+  httpd_path = get_option('httpd')[0]
+else
 httpd = find_program(get_option('httpd'), required: false)
 httpd_path = ''
 
@@ -100,6 +103,7 @@ if httpd.found()
     error('httpd version @0@ >= @1@: NO'.format(version, req_version))
   endif
 endif
+endif
 
 config_h.set_quoted('HTTPD_PROGRAM', httpd_path)
 
